packages/design-tokens/tokens/border-vars.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Now that we're moving the source of truth from json to ts, we could probably say goodbye to these tokens